forms/mainwindow.ui

```diff
@@ -214,9 +214,9 @@ border-bottom: 1px solid darkgrey;
            <property name="geometry">
             <rect>
              <x>0</x>
-             <y>0</y>
-             <width>741</width>
-             <height>835</height>
+             <y>-608</y>
```

Comment:
It's because I saved the .ui with the horizontal scroll bar of qt designer not at 0. It seems to only affect the way you see the .ui in qt designer because the real Johnny have the progress bar at 0 but I'll revert it for the same reason as your my comment on indexes.